platform/darwin/src/MGLAccountManager.m

```diff
@@ -1,26 +1,37 @@
 #import "MGLAccountManager_Private.h"
 #import "NSBundle+MGLAdditions.h"
+
 #if TARGET_OS_OSX
 #import "NSProcessInfo+MGLAdditions.h"
 #endif
 
 #if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
 #import "MGLMapboxEvents.h"
+#endif
 
-@interface MGLAccountManager ()
+@interface MGLSKUToken : NSObject
 
-@property (atomic) NSString *accessToken;
-@property (nonatomic) NSURL *apiBaseURL;
++ (NSString *)skuToken;
+
+@end
+
+@implementation MGLSKUToken
+
++ (NSString *)skuToken {
+    return @"not an actual token — this will be provided by the mbxaccounts library";
+}
 
 @end
-#else
+
 @interface MGLAccountManager ()
 
 @property (atomic) NSString *accessToken;
 @property (nonatomic) NSURL *apiBaseURL;
 
+@property (atomic) NSString *skuToken;
+@property (atomic) NSDate *skuTokenExpiration;
```

Comment:
These two lines (and their getters/setters) should also probably be guarded by `TARGET_OS_IPHONE` conditional compilation.

Reply:
Did this in https://github.com/mapbox/mapbox-gl-native/pull/14421/commits/4d1afd672ece9a2b3607208f96223ad515c61c0b, in part to fix [the macOS build](https://circleci.com/gh/mapbox/mapbox-gl-native/263284).